velox/connectors/hive/HiveDataSource.cpp

```diff
@@ -605,7 +605,11 @@ std::optional<RowVectorPtr> HiveDataSource::next(
     // wrapping the results.
     BufferPtr remainingIndices;
     if (remainingFilterExprSet_) {
+      auto filterStartMicros = getCurrentTimeMicro();
```

Comment:
nit: Move this inside `evaluateRemainingFilter`?

Reply:
Updated, thanks.